Packs/Jira/Integrations/JiraV2/JiraV2.yml

```
@@ -836,25 +527,48 @@ script:
     - contextPath: Ticket.Transitions.transitions
       description: A list of all possible transitions.
       type: Unknown
-  - deprecated: false
-    description: Available from Cortex XSOAR version 6.1.0. This command queries for
-      incidents that were modified since the last update. This method is only used
-      for debugging.
-    execution: false
+  - description: Available from Cortex XSOAR version 6.1.0. This command queries for incidents that were modified since the last update. This method is only used for debugging.
     name: get-modified-remote-data
-  dockerimage: demisto/oauthlib:1.0.0.48802
-  feed: false
+    arguments: []
+  - arguments:
+    - description: The ID of the issue to edit.
+      name: issueId
+      required: true
+      type: textArea
+    - description: The name of the assignee. Relevant for Jira Server only
+      name: assignee
+      required: true
+      type: textArea
+    description: Change assignee using dedicated endpoint for it
+    name: jira-issue-assign
+    outputs:
+    - contextPath: Ticket.Id
+      description: The ticket ID.
+      type: Unknown
```

```suggestion
      type: String
```

please add for each context path the correct `type`- https://xsoar.pan.dev/docs/integrations/context-and-outputs